app/dashboard/models.py

```diff
@@ -315,6 +315,10 @@ def value_in_usdt(self):
         except Exception:
             return None
 
+    @property
+    def token_value_in_usdt(self):
+        return round(convert_token_to_usdt(self.token_name), 2)
```

Comment:
doesnt the `value_in_usdt` method already do this?

Reply:
It gets the total value of USD  for the contract. This the USD value of the token and not the contract.

Reply:
ah right!